integration-tests/oauth2-reactive/src/main/java/com/okta/spring/tests/oauth2/reactive/implicit/ReactiveImplicitFlowApplication.java

```diff
@@ -48,8 +49,11 @@ public static void main(String[] args) {
 
         @Bean
         public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http) {
-            return http
-                .authorizeExchange()
+
+            Okta.configureResourceServer401ResponseBody(http);
```

Comment:
@dogeared @mraible any better?

Reply:
re: https://github.com/okta/okta-spring-boot/pull/121/files#r266094025

Reply:
well, since it's my suggestion, I approve! ;)